### DISCUSSION TITLE
missing FIT sensor

### DISCUSSION THREAD

**Discussion by hemiceni** [amberelectric/public-api#169] [Home Assistant]
New  solar and battery system, and new to amber. I have setup the amber integration in HA which gives me 5 sensors including home_general_price, and home_general_forecast - but there is no FIT nor FIT forecast sensor. What is the normal way to get this data in HA?
Thanks

**Comment by madpilot** [comment]
The FiT sensor will only show if your meter is configured for export.

Toggling the "show FiT" toggle in the app doesn't affect the API.

If you do have an export channel, can see you feed in in the app, but it's still not showing in the API, email info@amber.com.au and we can have a look.

**Comment by hemiceni** [comment]
ok. I know the electrician installing the system turned off exporting while I wait for a smart meter. So maybe it's that. At least I know now that the HA integration should have it when it's available.
(I already knew I cannot export yet, but was still hoping to find it in HA. There's plenty of work I could do in the meantime with automations etc)
